core/uemis-downloader.c

```diff
@@ -773,11 +773,11 @@ static bool uemis_delete_dive(device_data_t *devdata, uint32_t diveid)
 	return false;
 }
 
-/* This function is called for both divelog and dive information that we get
- * from the SDA (what an insane design, btw). The object_id in the divelog
```

Comment:
All changes from here until the end are to comments or debug output.
I am not a huge fan of script based search an replace changes like this...
This won't break anything - but it has the potential to break things (I do actually have tooling that parses the debug output, it just doesn't happen to trigger on the text that you changed)